pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConnection.java

```diff
@@ -524,6 +524,8 @@ protected void handleLookup(CommandLookupTopic lookup) {
 
     ClientConfigurationData createClientConfiguration() {
         ClientConfigurationData clientConf = new ClientConfigurationData();
+        /** The proxy service does not need to automatically clean up invalid connections, so set false. **/
+        clientConf.setAutoReleaseIdleConnectionsEnabled(false);
```

Comment:
@codelipenghui 
This prevents the proxy from automatically releasing the connection.